src/many-migration/src/lib.rs

```diff
@@ -559,9 +589,14 @@ impl<'a, T, E> MigrationSet<'a, T, E> {
     }
 
     #[inline]
-    pub fn update_at_height(&mut self, storage: &mut T, block_height: u64) -> Result<(), E> {
+    pub fn update_at_height(
+        &mut self,
+        storage: &mut T,
+        block_height: u64,
+        path: P,
```

Comment:
We should discuss this, I'd like to know what other ways we have. It adds a lot of complexity at the call sites. Best case, we don't need to change this method's signature, but in case we have to, this should probably be `Option<P>` considering how it's used with hardcoded `""` sometimes.